tools/tpm_manufacture.sh

```diff
@@ -9,6 +9,7 @@ CA_SRL=$KNC/ca.srl
 EK_PUB=$KNC/ek_pub.pem
 EK_CSR=$KNC/ek.csr
 EK_CERT=$KNC/ek_cert.der
+EK_CTX=$KNC/ek_ctx.der
```

Comment:
This file is not removed. On the other hand, `tpm2 createek (...) -c /dev/null` also works and except for one used transient handle (cleared later anyway) doesn't have any bad side effects.

Reply:
Fixed.